File: GridView/VirtualRadGridView/VirtualRadGridViewSortingFilteringAfterQ3_2015/VirtualRadGridViewSortingFilteringAfterQ3_2015VB/VirtualRadGridViewSortingFiltering.Core/VirtualMasterGridViewTemplate.vb

Imports System.Collections.Generic
Imports System.Text
Imports Telerik.WinControls.UI

Namespace VirtualRadGridViewSortingFiltering.Core
    Public Class VirtualMasterGridViewTemplate
        Inherits MasterGridViewTemplate
        Public Sub New()
            Me.ThrowExceptionOnDataOperationInVirtualMode = False
        End Sub
        Protected Overrides Function CreateListSource() As GridViewListSource
            Return New VirtualGridViewListSource(Me)
        End Function
    End Class
End Namespace

'=======================================================
'Service provided by Telerik (www.telerik.com)
'Conversion powered by NRefactory.
'Twitter: @telerik
'Facebook: facebook.com/telerik
'=======================================================

File: GridView/VirtualRadGridView/VirtualRadGridViewCS/VirtualRadGridViewSortingFiltering.Core/VirtualMasterGridViewTemplate.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls.UI;

namespace VirtualRadGridViewSortingFiltering.Core
{
    public class VirtualMasterGridViewTemplate : MasterGridViewTemplate
    {
        protected override GridViewListSource CreateListSource()
        {
            return new VirtualGridViewListSource(this);
        }
    }
}